main.go

```diff
@@ -83,7 +88,16 @@ func main() {
 	if uErr != nil {
 		panic(uErr)
 	}
+	// Open comments cmd
+	if input == "comments" {
+		frontpageID := 8863
+		commentURL := fmt.Sprintf("https://news.ycombinator.com/item?id=%v", frontpageID)
 
+		err := openLink(commentURL)
+		if err != nil {
+			panic(err)
+		}
```

Comment:
A common go pattern for dealing with errors:
```suggestion
		if err := openLink(commentURL); err != nil {
			panic(err)
		}
```